website/source/docs/providers/docker/r/image.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: "docker"
+page_title: "Docker: docker_image"
+sidebar_current: "docs-docker-resource-image"
+description: |-
+  Downloads and exports the ID of a Docker image.
+---
+
+# docker\_image
+
+Downloads and exports the ID of a Docker image. This can be used alongside
+[docker\_container](/docs/providers/docker/r/container.html)
+to programmatically get the latest image IDs without having to hardcode
+them.
+
+## Example Usage
+
+```
```

Comment:
If you specify these as `javascript` type, they will get syntax highlighting classes. We don't actually have attached styles iirc, but it would be easier to add syntax highlighting in the future this way.